app/code/core/Mage/Contacts/etc/system.xml

```diff
@@ -50,34 +50,62 @@
                     <show_in_website>1</show_in_website>
                     <show_in_store>1</show_in_store>
                     <fields>
+                        <sender_email_identity translate="label">
+                            <label>Email Sender</label>
+                            <frontend_type>select</frontend_type>
+                            <source_model>adminhtml/system_config_source_email_identity</source_model>
+                            <sort_order>10</sort_order>
+                            <show_in_default>1</show_in_default>
+                            <show_in_website>1</show_in_website>
+                            <show_in_store>1</show_in_store>
+                        </sender_email_identity>
                         <recipient_email translate="label">
                             <label>Send Emails To</label>
                             <validate>validate-email required-entry</validate>
-                            <sort_order>10</sort_order>
+                            <sort_order>20</sort_order>
                             <show_in_default>1</show_in_default>
                             <show_in_website>1</show_in_website>
                             <show_in_store>1</show_in_store>
                         </recipient_email>
-                        <sender_email_identity translate="label">
-                            <label>Email Sender</label>
+                        <email_template translate="label">
+                            <label>Email Template</label>
                             <frontend_type>select</frontend_type>
-                            <source_model>adminhtml/system_config_source_email_identity</source_model>
-                            <sort_order>20</sort_order>
+                            <source_model>adminhtml/system_config_source_email_template</source_model>
+                            <sort_order>30</sort_order>
                             <show_in_default>1</show_in_default>
                             <show_in_website>1</show_in_website>
                             <show_in_store>1</show_in_store>
-                        </sender_email_identity>
+                        </email_template>
+                    </fields>
+                </email>
+                <auto_reply_to_customer translate="label">
+                    <label>Auto Reply to Customer</label>
+                    <sort_order>50</sort_order>
```

Comment:
"Auto Reply to Customer" is used incorrectly. When we set an auto reply, it is obvious that a message will be sent automatically to the person who contacts you. This person can be a customer, a potential customer or a simple visitor. 

"Auto Reply" is correct in my opinion. In addition, it is in the Contacts section and is self-explanatory